abci/example/kvstore/kvstore_test.go

```diff
@@ -238,35 +238,6 @@ func TestCheckTx(t *testing.T) {
 	}
 }
 
-func TestClientAssignLane(t *testing.T) {
```

Comment:
If it exists in the feature branch, let's keep it, it will be invisible as change once we merge back the feature branch here.